### PR TITLE
Game

### PR DESCRIPTION
**Pull request by stiakov**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  [3, 6, 9],
+  [1, 5, 9],
+  [3, 5, 7],
+];
+const human = new Player('Human', 'X');
+const computer = new Player('Computer', '0');
+
+// Array.prototype.sample = function () {
+//   return this[Math.floor(Math.random() * this.length)];
+// };
```
Comment (JamesOkunlade) [inline js/index.js]: Please delete this. Don't leave unused code as a comment in your code.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      }
+      if (joinMoves.length === 9) {
+        document.getElementById('round_score').innerHTML = 'Tie!';
+        return true;
+      }
+    }
+    return false;
+  };
+
+  function turn(cellId) {
```
Comment (JamesOkunlade) [inline js/index.js]: Always use the `ES6 arrow function`

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    human.setPlayerMoves(cellId);
+    document.getElementById(cellId).innerText = human.getPlayerMark();
+    document.getElementById(cellId).removeEventListener('click', turnClick);
+  }
+  function turnClick(cell) {
```
Comment (JamesOkunlade) [inline js/index.js]: Use `arrow function`

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello, devs ✋ Nice work on the Tic-Tac-Toe game 👏 😄 . There are some things you'll need to improve on though.

**This project follows the lesson about Factory Functions and The Module Pattern so the main goal is to put into practice those concepts along with the main concepts of Object-Oriented Programming.** 

- Separate your code into different files and modules say `display/dom-manager`, `player`, `gameboard` and `play` and use `es6 import/export` to require each file in the other.
- Keep all dom related tasks in the `display/dom-manager` and don't mix it with the game logic.
- Change all your classic functions to `ES6 arrow functions`.
  
**Fix those points and request for re-review.**

**Status: `Changes required`**

Happy coding 😎

**Review by AdnanAfsari — APPROVED** [review]
Hello @stiakov @misselliev ✋ 

Nice work, its a 👍 form me.

**Status : Approved**

Happy Coding

**Pull request merged**